content/applications/websites/livechat.rst

```diff
@@ -5,80 +5,287 @@
+by clicking :guilabel:`New`, and filling out the :guilabel:`Create Operators` pop-up form. When the
+form is complete, click :guilabel:`SAVE & CLOSE` (or :guilabel:`SAVE & NEW` for multiple record
+creations).
 
 .. note::
-   GeoIP, which refers to the process of finding a computer terminal’s geographical location by its
-   IP address, must be installed on your server. Otherwise, under *Channel Rules*, countries are
-   not taken into account.
+   Current operators can be edited (or removed) by clicking on their respective boxes in the
+   :guilabel:`Operators` tab, which reveals a separate :guilabel:`Open: Operators` pop-up. In that
+   pop-up, adjust any information as needed, and click :guilabel:`Save`, or click :guilabel:`Remove`
+   to remove that operator from the channel.
+
+Options
+-------
+
+The *Options* tab on the live chat channel details form contains the visual and text settings for
+the live chat window.
+
+Livechat button
+~~~~~~~~~~~~~~~
+
+The *Livechat Button* is the icon that appears in the bottom corner of the website.
+
+.. image:: livechat/chat-button.png
+   :align: center
+   :alt: View of an Odoo website emphasizing the livechat button.
+
+Change the text in the :guilabel:`Text of the Button` field to update the greeting displayed in the
+text bubble when the live chat button appears on the website.
+
+Change the :guilabel:`Livechat Button Color` by clicking a color bubble to open the color selection
+window. Click the refresh icon to the right of the color bubbles to reset the colors to the default
+selection.
+
+.. tip::
+   Color selection, for the button or header, can be made manually, or through RGB, HSL, or HEX code
+   selection from the pop-up color selection window that appears when those color bubbles are
+   selected. Different options will be available, depending on your operating system.
+
+Livechat window
+~~~~~~~~~~~~~~~
+
+The *Livechat Window* is the where the live chat conversation takes place.
+
+Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open the chat
+window. This message will appear as though it is sent by a live chat operator, and should be an
+invitation to continue the conversation.
+
+Edit the :guilabel:`Chat Input Placeholder` to alter the text that appears in the box where visitors
+will type their replies.
+
+The *Channel Header* is the colored bar at the top of the chat window. The :guilabel:`Channel Header
+color` can be changed following the same steps as the :guilabel:`Livechat Button Color` above.
+
+.. figure:: livechat/chat-window.png
+   :align: center
+
+   The Livechat Window with a purple header. The chat input placeholder reads "Ask Something..."
+
+Channel rules
+-------------
+
+The *Channel Rules* tab on the live chat channel details form determines when the *Live Chat Window*
+opens on the website, by configuring when a *URL Regex* action is triggered (e.g., a page visit).
```

Comment:
Please add guilabels to the items with active instruction here rather than italics